cabal.project

```diff
@@ -13,7 +13,7 @@ repository cardano-haskell-packages
 -- See CONTRIBUTING for information about these, including some Nix commands
 -- you need to run if you change them
 index-state:
-  , hackage.haskell.org 2023-04-29T00:00:00Z
```

Comment:
normally, I wouldn't suggest this, but given we want to get 8.0.0 out and we don't want any issues coming up, I'ld like to hold this one back a couple days until we get the tag because of the hackage bump.

Reply:
Instead of bumping the index state, I will duplicate the module I wanted to share.  This will be deduplicated again after the release tag.